src/test/java/com/es/phoneshop/model/product/ArrayListProductDaoTest.java

```diff
@@ -2,20 +2,41 @@
 
 import org.junit.Before;
 import org.junit.Test;
+import java.math.*;
+import java.util.*;
+
 
 import static org.junit.Assert.assertTrue;
 
 public class ArrayListProductDaoTest
 {
     private ProductDao productDao;
-
+    
     @Before
     public void setup() {
         productDao = new ArrayListProductDao();
     }
 
     @Test
     public void testFindProductsNoResults() {
-        assertTrue(productDao.findProducts().isEmpty());
+        assertTrue(!productDao.findProducts().isEmpty());
```

Comment:
Can be simplified to assertFalse?
assertFalse(productDao.findProducts().isEmpty())